### PR TITLE
Convert PEPs 261, 267, 325, 358, 361

### PR DESCRIPTION
**Pull request by Mariatta**
Continuation for https://github.com/python/peps/issues/4

### CODE REVIEW

File: pep-0261.txt [hunk 1]
```diff
@@ -5,280 +5,287 @@ Last-Modified: $Date$
...
-    Unicode string
-
-          A Python type representing a sequence of code points with
-          "string semantics" (e.g. case conversions, regular
-          expression compatibility, etc.) Constructed with the 
-          unicode() function.
+========
+
+Character
+   Used by itself, means the addressable units of a Python
```
Comment (brettcannon) [inline pep-0261.txt]: How does this indented stuff look rendered? Should these be unindented and made into sub-sections?

Comment (Mariatta) [inline pep-0261.txt]: This becomes a `definition list` item: http://cpython-devguide.readthedocs.io/documenting.html?highlight=definition%20list#lists-and-quotes

But I can convert these into sub-headings too.

screen shot 2017-02-10 at 11 07 07 am

Comment (brettcannon) [inline pep-0261.txt]: That looks reasonable to me!

**Comment by brettcannon** [comment]
LGTM

File: pep-0261.txt [hunk 2]
```diff
@@ -5,280 +5,287 @@ Last-Modified: $Date$
...
+  raise ``ValueError``.
 
-        ISSUE 
+  ISSUE
 
-            Python currently allows \U literals that cannot be
-            represented as a single Python character. It generates two
-            Python characters known as a "surrogate pair". Should this
-            be disallowed on future narrow Python builds?
+     Python currently allows \U literals that cannot be
```
Comment (Mariatta) [inline pep-0261.txt]: Just noticed that the `\U` isn't rendered correctly 😅 

Will make another commit.

Comment (brettcannon) [inline pep-0261.txt]: Still LGTM 😄
